roles/common/tasks/rhel-entitlements.yml

```diff
@@ -14,26 +14,54 @@
   changed_when: false
   no_log: true
 
+- name: Set rhsm_registered
+  set_fact:
+    rhsm_registered: "{{ subscription.rc == 0 }}"
+  when: subscription is defined
```

Comment:
I believe this is not needed as ``subscription`` looks to always be defined, right?